docs/likelihood.rst

```diff
@@ -194,16 +194,20 @@ MC Statistical Uncertainty (staterror)
 
 As the sample counts are often derived from Monte Carlo (MC) datasets, they
 necessarily carry an uncertainty due to the finite sample size of the datasets.
-As explained in detail in :cite:`likelihood-Cranmer:1456844`, adding uncertainties for
+As explained in detail in :cite:`likelihood-Cranmer:1456844`, adding uncertainties for
 each sample would yield a very large number of nuisance parameters with limited
-utility. Therefore a set of bin-wise scale factors :math:`\gamma_b` is
+utility.
+Therefore a set of bin-wise scale factors :math:`\gamma_{cb}` is
```

Comment:
@kratsg @lukasheinrich I need to refresh myself on this, but if we clarify here that staterror is a per-channel modifier, then why don't we need to additionally denote this in the [Modifiers and Constraints table](https://scikit-hep.org/pyhf/intro.html#tab-modifiers-and-constraints)?

![modifiers](https://user-images.githubusercontent.com/5142394/166620849-5b3d6f68-1d3b-4861-ae85-add3c2d749fe.png)

Did we have a good reason for this?

Reply:
I think in the end per-bin implies per-channel, since different channels may not have the same binning.

Reply:
> I think in the end per-bin implies per-channel, since different channels may not have the same binning.

Yeah, this is somewhat my point. Though while per-bin notation should imply per-channel, one could also say that being overly explicit probably wouldn't hurt or new users and I don't think would confuse people experienced with the spec.

I could go either way here, so would be good to get @lukasheinrich to weight in given that he is the only one who hasn't yet.

Reply:
> Yeah, this is somewhat my point. Though while per-bin notation should imply per-channel, one could also say that being overly explicit probably wouldn't hurt or new users and I don't think would confuse people experienced with the spec.

This probably depends on how we define the indexing right? Arguably, one could say that `(c,b) = (2, 0)` is equivalent to the flattened coordinate `(b,) = (10,)`. So the notation of `sigma_{cb}` should be equivalent to `sigma_{b}` but the `b` have different interpretations.